contracts/examples/Mars.sol

```diff
@@ -91,14 +104,36 @@ contract Mars is IbcReceiverBase, IbcReceiver {
         dispatcher.channelCloseInit(channelId);
     }
 
+    /**
+     * @dev Sends a packet with a greeting message over a specified channel.
+     * @param message The greeting message to be sent.
+     * @param channelId The ID of the channel to send the packet to.
+     * @param timeoutTimestamp The timestamp at which the packet will expire if not received.
+     * @dev This method also returns sequence from the dispatcher for easy testing
+     */
+    function greet(string calldata message, bytes32 channelId, uint64 timeoutTimestamp)
```

Comment:
Does this need to be `payable` too?

Reply:
no, because greet should only be used when there isn't a fee. greetWithFee is what should be used if there needs to be a fee sent, and that is indeed payable. 

I added docs clarifying this since i'd imagine this might be a source of confusion in general

Reply:
yeah, that's what I thought too. note that it is marked as `payable` though - hence the question :D

Reply:
ahhh whoops didn't realize it was payable

Reply:
should be made nonpayable now!